src/components/common/MenuDrawer.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { ROUTES } from '@/constants/routes';
+import { cn } from '@/lib/utils';
+import Link from 'next/link';
+
+interface MenuDrawerProps {
+  isOpen: boolean;
+  onClose: () => void;
+  pathname: string;
+}
+
+interface MenuItemProps extends React.HTMLAttributes<HTMLDivElement> {
+  text: string;
+  selected?: boolean;
+}
+
+const MenuDrawer = ({ isOpen, onClose, pathname }: MenuDrawerProps) => {
+  return (
+    <ul
+      className={`fixed right-0 top-0 z-50 h-screen w-full bg-[#08072D80] shadow-lg transition-transform duration-300 ease-in-out backdrop-blur-20px lg:w-[30vw] ${
```

Comment:
```suggestion
className={cn(
        'fixed right-0 top-0 z-50 h-dvh w-full bg-[#08072D80] backdrop-blur-[20px] transition-transform duration-300 ease-in-out lg:w-[30vw]',
        isOpen ? 'translate-x-0' : 'translate-x-full',
      )}
```

cn 사용해서 위처럼 수정하면 좋을 것 같습니다! (쉼표로 연결)
그리고 모바일 접속 고려하여, `h-screen` 대신 `h-dvh` 로 수정하는게 좋을 것 같아요!

참고 : https://abcdqbbq.tistory.com/104

Reply:
감사합니다!